pressbooks/pressbooks-multi-institution
# feat: add unassigned filter for assign users list

Issue: https://github.com/pressbooks/pressbooks-multi-institution/issues/104

This PR:
- Centralized books/assign.blade.php and users/assign.blade.php templates.
- Adds the Unassigned link filter in Assign Users and Assign Books pages.

### Testing case
- As a Super Admin or NM go to Institutions > Assign Users page.
- You should see 2 new links at the top of the table: `All` and `Unassigned`. Both should display the total of users and the total number of unassigned users within the parenthesis of each filter.
- Click on the `Unassigned` filter. Only the unassigned users should be displayed in the table.
- Click on the `All` filter. All the users should be displayed in the table, regarding whether they are assigned or not to an institution.
- Repeat the process on the Institutions > Assign Books page.
